src/views/SpaceBoost.vue

```diff
@@ -0,0 +1,848 @@
+
+                    {{ ethFee }}
+                    ETH
+                  </span>
+                </div>
+              </div>
+            </TuneBlockFooter>
+          </TuneBlock>
+          <TuneBlock>
+            <template #header>
+              <TuneBlockHeader
+                title="Eligibility"
+                sub-title="Choose an option that best incentivises meaningful participation."
+              />
+            </template>
+
+            <TuneListbox
+              v-model="form.eligibility.choice"
+              :items="eligibilityOptions"
+              label="Eligible to"
+            />
+            <TuneBlockFooter v-if="bribeDisabled">
+              <BaseMessage level="info">
+                Selecting a specific choice is disabled for the
+                <span class="font-semibold">
+                  {{ space.name }}
+                </span>
+                space
+              </BaseMessage>
+            </TuneBlockFooter>
+          </TuneBlock>
+
+          <TuneBlock>
+            <template #header>
+              <TuneBlockHeader
+                title="Distribution"
+                sub-title="Define how the reward pool is distributed to eligible voters."
+              />
+            </template>
+            <div class="space-y-3">
+              <div class="flex gap-[12px]">
+                <div
+                  v-for="item in DISTRIBUTION_TYPE_ITEMS"
+                  :key="item.value"
+                  class="rounded-xl p-3 cursor-pointer border w-1/2"
+                  :class="{
+                    'border-skin-link': form.distribution.type === item.value
+                  }"
+                  @click="form.distribution.type = item.value"
+                >
+                  <div
+                    class="flex justify-between font-semibold text-skin-heading"
+                  >
+                    {{ item.name }}
+                    <i-ho-check v-if="form.distribution.type === item.value" />
+                  </div>
+                  <div class="leading-[18px] pr-4">
+                    {{ item.description }}
+                  </div>
+                </div>
+              </div>
+              <template v-if="form.distribution.type === 'weighted'">
+                <TuneSwitch
+                  v-model="form.distribution.hasWeightedLimit"
+                  label="Define a maximum reward per voter"
+                />
+
+                <TuneInput
+                  v-model="form.distribution.weightedLimit"
+                  label="Max reward"
+                  type="number"
+                  placeholder="100"
+                  always-show-error
+                  :error="formErrorMessages?.weightedLimit"
+                  @click="form.distribution.hasWeightedLimit = true"
+                >
+                  <template #after>
+                    <div class="-mr-[8px]">{{ selectedToken?.symbol }}</div>
+                  </template>
+                </TuneInput>
```

Comment:
The input is overflowing on the after slot 

![Screenshot 2024-03-07 at 01 51 58](https://github.com/snapshot-labs/snapshot/assets/495709/4249232f-3bdd-4081-abfa-382d3ac78c28)